.github/workflows/php.yml

```diff
@@ -30,7 +30,7 @@ jobs:
                   coverage: none
 
             - name: Set up branches
-              run: git checkout -b master refs/remotes/origin/master && git checkout -
```

Comment:
I couldn't run linter without this change.

Reply:
Did you happen to see [this post](p6rkRX-39Y-p2) that Gabriel wrote? Could it be related to that?

Reply:
I did, but I've made the branch before https://github.com/Automattic/sensei/pull/4956/files changes were merged to trunk I think that was the reason why I didn't have them in my branch. I will manually fix the conflicts in yml once the other PR ([update/fix_sorting_for_students_column](https://github.com/Automattic/sensei/tree/update/fix_sorting_for_students_column) is merged in the trunk